spec/controllers/public_pages_controller_spec.rb

```diff
@@ -83,33 +83,4 @@
       expect(response).to be_ok
     end
   end
-
-  describe "#maintenance" do
-    it "renders even when in maintenance mode" do
-      ENV['MAINTENANCE_MODE'] = '1'
-      get :maintenance
-      ENV.delete('MAINTENANCE_MODE')
-
-      expect(response).to be_successful
-    end
-  end
-
-  describe '#at_capacity' do
-    it 'renders when at capacity (in at_capacity mode)' do
-      ENV['AT_CAPACITY'] = '1'
-      get :at_capacity
-      ENV.delete('AT_CAPACITY')
-
-      expect(response).to be_successful
-    end
-
-    it 'renders when not at capacity (in at_capacity mode)' do
-      # TODO: should this even render when not at capacity, or should
-      # it redirect to somewhere else?
-
-      get :at_capacity
-
-      expect(response).to be_successful
-    end
-  end
```

Comment:
Not clear on the reasons for deleting these tests. Do we have any other tests that check that these pages exist and can be rendered by the controller? Non-blocking question.